src/mctpd.c

```diff
@@ -2668,6 +3004,36 @@ static int setup_bus(ctx *ctx)
 		goto out;
 	}
 
+	rc = sd_bus_match_signal(
+		ctx->bus,
+		NULL,
+		NULL,
+		"/xyz/openbmc_project/inventory",
+		"org.freedesktop.DBus.ObjectManager",
+		"InterfacesAdded",
+		on_interface_added,
+		ctx
+	);
```

Comment:
I have tried `arg[0, ...]` and `arg0namespace` but didn't work well. Do you have any idea how to filter it?
I also checked how others OpenBMC's code handle it. They use callback function to filter the signal instead of filtering signals by match.

Reply:
Ah, I had incorrectly assumed that `arg0` would specify the interface name, but it's actually the object path. That may make it difficult to filter. I'll see if there's a better alternative, but we may just have to rely on filtering in the callback (as you have done already).